plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/form/field/Base.scss

```diff
@@ -8,10 +8,25 @@
  * cls: 'x-form-readonly'
  */
 
-.x-form-field[readonly].x-form-field:not([role="combobox"]) {
+@mixin nx-form-field-readonly {
   color: $color-dark-gray;
-  background-color: $color-smoke;
```

Comment:
Depends on what we want these fields to look like. The only readonly fields I am aware of are on this page, and they all also apply nx-no-border which immediately overwrites this background. If we want to be consistent with readonly fields on other forms, they should also use the nx-no-border styling, so we might want to just merge the two rules here instead.